src/adapter/src/catalog/state.rs

```diff
@@ -832,7 +832,7 @@ impl CatalogState {
     }
 
     /// Parses the given SQL string into a pair of [`Plan`] and a [`ResolvedIds)`.
-    #[tracing::instrument(level = "info", skip(self, pcx))]
+    #[tracing::instrument(level = "info", skip(self, pcx, catalog))]
```

Comment:
This will trace the full `create_sql` which isn't redacted. We've been using `skip_all` elsewhere. Also you mention turning on debug, but this is tracing at info (my suggestion is also info, the default). I would think this function was getting traced by default since info is on most places now? A bit scary.

```suggestion
    #[tracing::instrument(skip_all)]
```

Reply:
> Also you mention turning on debug, but this is tracing at info

Yeah I was confused by that, I only saw this when turning on debug logging, I didn't see it traced at info 🤷